egs/csj/s5/local/nnet3/run_tdnn_swbd.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
I would recommend keeping only one local/nnet3/run_tdnn.sh with the recommended setting. The current setup having multiple tdnn scripts is confusing.